nibabies/workflows/anatomical/base.py

```diff
@@ -1,32 +1,38 @@
 """Base anatomical preprocessing."""
+import warnings
+from pathlib import Path
+from typing import Literal, Optional, Union
+
 from nipype.interfaces import utility as niu
 from nipype.pipeline import engine as pe
+from niworkflows.engine.workflows import LiterateWorkflow
+from niworkflows.utils.spaces import Reference, SpatialReferences
```

Comment:
This can be an expensive import. I might use a `from __future__ import annotations` to avoid runtime resolution of annotations, and then put this in an `if ty.TYPE_CHECKING:` block.